scripts/avocado

```diff
@@ -56,10 +56,6 @@ def handle_exception(*exc_info):
     sys.exit(-1)
 
 
-if __name__ == '__main__':
-    sys.excepthook = handle_exception
```

Comment:
The point is that when avocado import fails we do want to store the info about this failure. If we moved the import before setting the excepthook we won't get the treatment on import error.